### DISCUSSION TITLE
"Method Not Implemented" raises when confirmation email is trying to be sent.

### DISCUSSION THREAD

**Discussion by YOCKOW** [bluesky-social/atproto#1750] [Clarifications and "Why?"]
After a PDS server started running and the first account was created with app.bsky-sandbox.dev,  I tried to send confirmation email by clicking "Verify My Email". However, the result is "Method Not Implemented" error.

I use [Google's SMTP relay service](https://support.google.com/a/answer/2956491?hl=en) without username/password (using "Only accept mail from the specified IP addresses" authentication). So the PDS env file is like below:

```env
PDS_EMAIL_FROM_ADDRESS="no-reply@<my domain here>"
PDS_EMAIL_SMTP_URL="smtp://smtp-relay.gmail.com:587/"
```

I made sure some mails were sent by another service with the same configurations.
What am I missing?

-----
PDS {"version":"0.3.0-beta.3"}

**Comment by bnewbold** [comment]
I suspect that the sandbox PDS distribution may be out-of-date compared to what is running in production, and the sandbox distribution may not include email verification (yet). I know there has been some work in progress to narrow the gap between current PDS development and the sandbox distribution, but I don't know if that is complete yet.

@dholms could you confirm?
